hs_file_types/models/base.py

```diff
@@ -702,6 +702,12 @@ def get_preview_data_url(self, resource, folder_path):
         # - subclass needs to override this
         return None
 
+    @classmethod
```

Comment:
This method is no longer used and should be deleted

Reply:
I’ve removed this method here: https://github.com/hydroshare/hydroshare/pull/4311/commits/88b583f6fac4113fe7448ffb38aa2e57ec97714e